src/main/java/org/mockbukkit/mockbukkit/inventory/InventoryMock.java

```diff
@@ -319,7 +319,7 @@ public ItemStack addItem(@NotNull ItemStack item)
 	@Override
 	public ItemStack @NotNull [] getContents()
 	{
-		return items;
+		return items.clone();
```

Comment:
Should this not also return a clone of each element in the array?

Reply:
I would do this:
```java
Arrays.toStream(items).map(ItemStack::clone).toArray(ItemStack[]::new)
```

Reply:
ok got it